src/site/content/en/index.njk

```diff
@@ -110,13 +110,21 @@ date: 2018-11-05
     </a>
   </section> #}
 
-  <section class="w-cds-promo">
+  <section class="w-cds-promo" id="cds">
     <div class="w-cds-promo__grid">
       <div>
-        <img
-          src="/images/modules/cds19-lockup-stack.svg"
-          alt="chrome dev summit 2019"
-        >
+        <div class="w-cds-promo__video">
+          <div class="w-youtube">
+            <iframe
```

Comment:
Simple question, do we not want to use the custom element @paulirish put out to keep the ifram from loading? I know we're lazy loading it, but wouldn't it still have a perf hit on lci?